packages/shared/src/rpc/index.ts

```diff
@@ -268,3 +274,24 @@ export const getPoolPriceV2 = createRequest(
 );
 
 export const getBlockHash = createRequest('chain_getBlockHash', hexString);
+
+const boostPoolsDepthResponseSchema = z.array(
+  z.intersection(
+    rpcAssetSchema,
+    z
+      .object({
+        tier: number,
+        available_amount: u128,
+      })
+      .transform((obj) => ({
+        tier: obj.tier,
+        availableAmount: obj.available_amount,
```

Comment:
this is automatically taken care of
```suggestion
```

Reply:
Not sure why but the response properties are not being camel cased for some reason.

Reply:
Ah, i see the issue. The response is an array of objects and we don't handle that properly in the `camelCaseKeys` method.

Reply:
Fixed it